README.md

```diff
@@ -1,4 +1,4 @@
-# Lightweight data provider (1.1.2)
+# Lightweight data provider (1.1.3)
```

Comment:
Maybe update minor version because we add 2 new features

Reply:
Version 1.2 is reserved for Cradle with groups